cost-model/src/transaction_cost.rs

```diff
@@ -67,11 +67,80 @@ impl TransactionCost {
     }
 
     pub fn sum(&self) -> u64 {
-        self.signature_cost
-            .saturating_add(self.write_lock_cost)
-            .saturating_add(self.data_bytes_cost)
-            .saturating_add(self.builtins_execution_cost)
-            .saturating_add(self.bpf_execution_cost)
-            .saturating_add(self.loaded_accounts_data_size_cost)
```

Comment:
i understand dropping the bpf execution cost 'cause native, but why loaded accounts?  votes still load accounts. is the cost misnamed and actually reflects the bpf marshaling of account data?

Reply:
The issue is vote tx does not come with compute_budget instruction to set loaded accounts size limit, so when it starts to count accounts size to CU (behind feature gate), the default 64MB will be assumed for such tx, that'd add ~16K CU to simple vote, make it hits vote block limits much quicker, reducing number of votes per block.

One option is to require vote to use compute budget to set limits, which could be larger change, potentially introducing security concerns (for example, one can send votes with max bytes, taking up block space, censoring other votes).

As a bug fix, this PR keeps vote CU unchanged regardless of mentioned feature gate. 

For longer term, have another PR #33230 to separate votes. At the back of my head, thinking to give vote a separate path from cost model, scheduler and perhaps fee calculation too. If that approach lands, then we can discuss how vote should be priced separately.

Reply:
ah! can we assign a more accurate static value in that case?  we know what a simple vote looks like. better to price accurately than undercount and have to charge more later 😅

Reply:
yea, would be nice to have static value for vote transaction. But so far (in all currently deployed version at all clusters) vote tx cost varies on how many signatures there are (1 or 2), therefor number of write locks, and the ix's data size (eg `vec<slot>`). This is captured by ln71-74 below, and in test at ln135.

In the near future, after activating the feature gate #30657, the loaded accounts size will be added to tx's cost; later when transaction's usage cost are being further enhanced, more factors will contribute to normal transaction's cost. This pr is to keep vote tx's cost unchanged then, shield votes packing from other changes to minimize chances of impacting consensus. 

But it does undercount vote cost -- the two writable accounts' size are not accounted for. Counting writable accounts' size is tricky for vote tx, since we don't really want to include compute-budget instructions in simple vote. So it'd be easier to assign the vote tx a static cost, such as the one defined at Ln 135 in test. But I haven't checked ledger to see if there are cases that a vote's node or vote accounts are exceptional large, therefore vote's cost are currently being significantly undercounted, or will it be if assigning vote a static cost.  

In any case, this perhaps warrants a separate discussion, maybe at #33230?

Reply:
PS. also want to keep it simple enough to be backport-able 😃